server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -182,6 +183,7 @@ impl ChangelogTableName {
             ChangelogTableName::BundledItem => ChangeLogSyncStyle::Central,
             ChangelogTableName::ContactForm => ChangeLogSyncStyle::RemoteToCentral,
             ChangelogTableName::SystemLog => ChangeLogSyncStyle::RemoteToCentral, // System Log records won't be synced to remote site on initialisation
+            ChangelogTableName::InsuranceProvider => ChangeLogSyncStyle::Legacy,
```

Comment:
Technically we don't need changelog for this, however I figure it doesn't hurt and came free with Andrei's snippet